### ISSUE TITLE
Version information printed to stderr

### ISSUE THREAD

**Issue by don-philipe** [prometheus/statsd_exporter#466]
Hello,

when calling `statsd_exporter --version` the output is printed to `stderr` instead of `stdout`. This can be verified by:

```sh
statsd_exporter --version 2> /dev/null
```

which prints nothing on console.

I guess that's not intended? It's kinda annoying when using that command with ansible. Other exporters like `node_exporter` print version information to `stdout`. I would love to see this behaviour for `statsd_exporter` as well :pray: 

statsd_exporter version: 0.22.8

**Comment by matthiasr** [comment]
Indeed, if the user explicitly asks for information it should go to stdout, and we should be in line with the other official exporters. PRs welcome!

**Comment by don-philipe** [comment]
Help text is printed to `stderr` as well.

I opened #469 that will change the default behavior of the kingpin lib. That lib will print to `stderr` by default. To output text to `stderr` another writer needs to be explicitly set (like it is done at `node_exporter` [here](https://github.com/prometheus/node_exporter/blob/master/node_exporter.go#L174)).
